spec/forms/legal_aid_applications/used_multiple_delegated_functions_form_spec.rb

```diff
@@ -50,8 +44,6 @@
         end
 
         it 'updates the application types with no reported on date' do
```

Comment:
Should this description be changed to `updates the proccedings with no reported on date`?